### PR TITLE
Replaced load balancer and sub-items by generated code

### PR DESCRIPTION
**Pull request by MikhailTryakhov**

### CODE REVIEW

File: lib/commands/arm/network/LoadBalancers-backendAddressPools._js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+  backendAddressPools.command('list [resource-group] [lb-name]')
+    .description($('List backend address pools'))
+    .usage('[options] <resource-group> <lb-name>')
+    .option('-g, --resource-group <resource-group>', $('the resource group'))
+    .option('-l, --lb-name <lb-name>', $('the lb name'))
+    .option('-s, --subscription <subscription>', $('the subscription identifier'))
+    .execute(function(resourceGroup, lbName, options, _) {
+      var subscription = profile.current.getSubscription(options.subscription);
+      var networkManagementClient = utils.createNetworkManagementClient(subscription);
+
```
Comment (balajikris) [inline lib/commands/arm/network/LoadBalancers-backendAddressPools._js]: nit: unnecessary newline

File: lib/commands/arm/network/loadBalancers._js [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+      if (result) {
+        throw new Error(util.format($('load balancer with name "%s" already exists in the resource group "%s"'), name, resourceGroup));
+      }
+
+      var parameters = {};
+      if(options.location) {
+        parameters.location = options.location;
+      }
+      if(options.tags) {
+        if (utils.argHasValue(options.tags)) {
```
Comment (balajikris) [inline lib/commands/arm/network/loadBalancers._js]: `if(x && y)`

File: lib/commands/arm/network/loadBalancers._js [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+            row.cell($('Location'), item.location || '');
+            var resInfo = resourceUtils.getResourceInformation(item.id);
+            row.cell($('Resource group'), resInfo.resourceGroup);
+            row.cell($('Provisioning state'), item.provisioningState);
+          });
+        }
+      });
+    });
+
+
```
Comment (balajikris) [inline lib/commands/arm/network/loadBalancers._js]: extra newline

**Comment by MikhailTryakhov** [comment]
@balajikris this code block is regenerated and ready for merge

**Comment by balajikris** [comment]
@MikhailTryakhov - This PR now has conflicts, most likely from your other PRs, 2 of which I just merged. Can you please fix the merge conflicts here?

**Comment by MikhailTryakhov** [comment]
@balajikris this PR is ready

**Pull request merged**